recipe/meta.yaml

```
@@ -51,8 +61,12 @@ test:
     - sksurv.linear_model
   commands:
     - pip check
```

it uses `setuptools-scm`, it might be useful to add a test command:

```suggestion
    - pip check
    # check that pip gets the correct version 
    - python -c "from importlib.metadata import version; assert(version('{{ name }}')=='{{ version }}')"
```
if `importlib.metadata` cannot be found, then we remove it